src/plugins/kconfig/README.md

```diff
@@ -0,0 +1,56 @@
+- infos = Information about the kconfig plugin is in keys below
+- infos/author = Author Name <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/kconfig storage kconfig
```

Comment:
```suggestion
- infos/provides = storage/kconfig
```

"storage kconfig" should not needed, did you find some problem when its not there?